HospitalWebAppTests/UnitTest1.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using Xunit;
+
+namespace HospitalWebAppTests
+{
+    public class UnitTest1
+    {
+        [Fact]
+        public void Test1()
+        {
+        }
+    }
+}
```

Comment:
what these tests should represent and why they are empty

Reply:
Tests in this project should test .net web api and they are empty because we still don't have specific feature to test.